README.md

```diff
@@ -230,12 +230,14 @@ The optional options object can be passed in with any of the following propertie
 #### Dependencies
 
 ```
-  <!-- Bootstrap --> 
-  <link rel="stylesheet" href="../node_modules\bootstrap\dist\css\bootstrap.min.css">
   <script src="../node_modules\jquery\dist\jquery.min.js"></script>
   <script src="../node_modules\bootstrap\dist\js\bootstrap.min.js"></script>
+<<<<<<< HEAD
```

Comment:
Looks like this slipped through during a merge conflict fix?

Reply:
Oh yeah, it sure did. And it looks like there were some in there from before. I think I fixed it all.